libs/island-ui/core/src/lib/Tooltip/Tooltip.tsx

```diff
@@ -99,7 +87,7 @@ export const Tooltip: FC<TooltipProps> = ({
             height: `${iconSize}px`,
           }}
         >
-          <InfoIcon />
+          <Icon icon="informationCircle" color={color} />
```

Comment:
regarding `iconSize`, after this change won't we need to use the `Icon` `size` prop instead of setting width height?